addons/survey/static/src/js/survey_form.js

```diff
@@ -441,9 +441,20 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
         self.$(selectorsToFadeout.join(',')).fadeOut(this.fadeInOutDelay, function () {
             resolveFadeOut();
         });
+        // Background management - Fade in / out on each transition
+        $('div.o_survey_background').addClass('o_survey_background_transition');
 
         Promise.all([fadeOutPromise, nextScreenPromise]).then(function (results) {
-            return self._onNextScreenDone(results[1], options);
+            var nextScreenResult = results[1];
+            // always refresh, even if same background. If same, it will already be in cache anyway.
+            var nextQuestionBackground = nextScreenResult.background_image_url;
+            var preloadBackgroundPromise = self._preloadBackground(nextQuestionBackground);
+
+            return Promise.resolve(preloadBackgroundPromise).then(function (backgroundResults) {
+                $('div.o_survey_background').css("background-image", "url(" + nextQuestionBackground + ")");
```

Comment:
Linting: shouldn't "url(...)" be "url(...);" with a semicolon at the end?

Reply:
nope ! :smile: